Bk junit5 test

evanyeohboonkhai opened the pull request.

willardsm commented on java-core/src/main/java/ai/certifai/junit5/ex1/BasicMethodAssertion.java:
```diff
@@ -1,12 +1,12 @@
+package ai.certifai.junit5.ex1;
 
 /**
  * Demo of Junit5 Test (Class)
```
You can write some introduction and where do you put the test file, because beginner might dont know what is the purpose of this file.

willardsm commented on java-core/src/test/java/ai/certifai/junit5/ex2/GetSetMethodAssertionTestSolution.java:
```diff
@@ -0,0 +1,43 @@
+
+class GetSetMethodAssertionTestSolution {
+    @Test
+    void getCoffeeType() {
```
I suggest you to follow the convention on how people name their test case. you can refer to this link for more information: https://dzone.com/articles/7-popular-unit-test-naming

willardsm commented on java-core/src/main/java/ai/certifai/junit5/ex1/BasicMethodAssertion.java:
```diff
@@ -1,12 +1,12 @@
  */
 
-public class MathUtils {
+public class BasicMethodAssertion {
```
For the naming of this class you can change back to MathUtils, then the test class naming for this class you can name it TestMathUtils or MathUtilsTest. do the same for GetSetMethodAssertion, change back to Coffee, and the test class you can name it TestCoffee or CoffeeTest

willardsm commented on java-core/src/main/java/ai/certifai/junit5/ex1/BasicMethodAssertion.java:
```diff
@@ -1,12 +1,12 @@
-package solution;
+package ai.certifai.junit5.ex1;
```
the folder naming i suggest to change it to "unittest" instead of "junit5". unittest is more topical based and junit5 is the name of the library

willardsm commented on java-core/src/test/java/ai/certifai/junit5/ex1/BasicMethodAssertionTest.java:
```diff
@@ -0,0 +1,124 @@
+    @ParameterizedTest
+    @ValueSource(ints ={3,4,5,6})
+    void test(int number){
+        assertEquals(0,number%3);
```
I run the test cases and fail in this function, is it you make it fail on purpose?

willardsm commented on java-core/src/test/java/ai/certifai/junit5/ex1/BasicMethodAssertionTest.java:
```diff
@@ -0,0 +1,124 @@
+    @ParameterizedTest
+    @ValueSource(ints ={3,4,5,6})
+    void test(int number){
+        assertEquals(0,number%3);
```
I run the test cases and fail in this function, is it you make it fail on purpose?

evanyeohboonkhai replied:
![image](https://user-images.githubusercontent.com/59526258/105125058-7aa56580-5b16-11eb-9c9c-4f145669457f.png)
yes, it is demonstrated to the student it can test the element {3,4,5,6} 1 by 1

evanyeohboonkhai replied:
those who have a remainder of  0 will pass, those who do not will fail

willardsm replied:
oh, then you should put a proper naming, like the example in this post: https://dzone.com/articles/7-popular-unit-test-naming

evanyeohboonkhai commented:
@willardsm 
Few change has been made
1. changed package name to unitTest

2. changed class file name 
- BasicMethodAssertion -> MathUtils
- GetSetMethodAssertionTest -> Coffee 
-
3. changed the test file name 
- BasicMethodAssertionTest -> MathUtilsTest

4. removed GetSetMethodAssertionTestSolution test file, and the solution is commented at bottom of the Coffee class file

willardsm commented:
Done the review, this PR is ready to merge
